charts/linkerd2/values.yaml

```diff
@@ -241,6 +241,10 @@ installNamespace: true
 nodeSelector:
   beta.kubernetes.io/os: linux
 
+# Tolerations settings here, would affect on all linkerd deployments
```

Comment:
nit: this comment could also be that of similar to `# nodeSelector` above

Reply:
updated in [2611c65](https://github.com/linkerd/linkerd2/pull/4626/commits/2611c65776aa8063bb87d73054ca6356574126e6)